ts/packages/ts/src/commands/commands.test.ts

```diff
@@ -221,5 +243,147 @@ describe(`commands`, () => {
         await Promise.all(deletePromises);
       }
     });
+    describe('Tests for lazy configs', () => {
+      test('run EvalCommand without configs but  `cfgu.lazy && cfgu.required = true`', async () => {
+        expect.assertions(1);
+        await expect(() =>
+          new EvalCommand({
+            store: store1,
+            set: set1,
+            schema: new ConfigSchema('lazy', {
+              K1: {
+                type: 'String',
+                lazy: true,
+                required: true,
+              },
+            }),
+          }).run(),
+        ).rejects.toBeInstanceOf(ConfigError);
+      });
+      test('run EvalCommand without override for lazy config when `cfgu.required = true`', async () => {
+        expect.assertions(1);
+        await expect(() =>
+          new EvalCommand({
+            store: store1,
+            set: set1,
+            schema: new ConfigSchema('lazy', {
+              K1: {
+                type: 'String',
+                lazy: true,
+                required: true,
+              },
+              K2: {
+                type: 'String',
+              },
+            }),
+            configs: {
+              K2: '2',
+            },
+          }).run(),
+        ).rejects.toBeInstanceOf(ConfigError);
+      });
+      test('EvalCommand with override for lazy config when `cfgu.required = false`', async () => {
+        expect.assertions(1);
+        await expect(() =>
+          new EvalCommand({
+            store: store1,
+            set: set1,
+            schema: new ConfigSchema('lazy', {
+              K1: {
+                type: 'String',
+                lazy: true,
+                required: true,
```

Comment:
the test description says `cfgu.required = false`

Reply:
The "error" here is in the title not the prop.
the test right below has this prop = false.

Thanks